microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -3,187 +3,219 @@ description: Microsoft Edge Privacy Whitepaper
 
 On other platforms, to generate a new resettable identifier \(ID\) that is unique to your browser, navigate to `edge://settings/privacy` and turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The reset \(ID\) functionality may be different for devices managed with group policies set by your organization. 
 
 If you're using Windows 10 version 1803 (April 2018 Update) or later, to view product data shared with Microsoft in the Diagnostic Data Viewer, navigate to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback** and select **Open Diagnostic Data Viewer** under **View diagnostic data**. 
 
 For other platforms or versions of Windows 10 version 1803 and earlier, navigate to `edge://data-viewer` to view diagnostic data. To view the data that's periodically sent to Microsoft since the last time the viewer was opened, navigate to `edge://data-viewer`. To see what data has been sent to Microsoft for your specific session, refresh the viewer. The data used to populate `edge://data-viewer` is stored locally on the device. To clear the data in the viewer, close the `edge://data-viewer` tab. 
 
-To help us improve Microsoft products and services, de-identified and aggregated diagnostic data is stored for up to two years. Because the diagnostic data is not collected from or stored with your Microsoft account, the diagnostic data may not be viewed or deleted from your [Microsoft privacy dashboard][MicrosoftAccountPrivacy]. To delete the diagnostic data on Windows 10, navigate to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback** and select **Delete** under **Delete diagnostic data**. The delete diagnostic data functionality is only supported on Windows 10 version 1803 or later. For more information, navigate to [Diagnostics, feedback, and privacy in Windows 10][MicrosoftSupport4468236]. 
+To help us improve Microsoft products and services, anonymous and aggregated diagnostic data is stored for up to two years. Because the diagnostic data is not collected from or stored with your Microsoft account, the diagnostic data may not be viewed or deleted from your [Microsoft privacy dashboard][MicrosoftAccountPrivacy]. To delete the diagnostic data on Windows 10 and later, navigate to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback** and select **Delete** under **Delete diagnostic data**. The delete diagnostic data functionality is only supported on Windows 10 version 1803 or later. For more information, navigate to [Diagnostics, feedback, and privacy in Windows 10 and later][MicrosoftSupport4468236]. 
```

Comment:
we should not be using the term "anonymous".  We can change de-identified to "we remove personal identifiers..." or some variant like that.